ambry-frontend/src/main/java/com/github/ambry/frontend/AmbryIdConverterFactory.java

```diff
@@ -192,7 +192,7 @@ private CompletionStage<String> convertId(String input, RestRequest restRequest,
         long expirationTimeMs =
             Utils.addSecondsToEpochTime(properties.getCreationTimeInMs(), properties.getTimeToLiveInSeconds());
         NamedBlobRecord record = new NamedBlobRecord(namedBlobPath.getAccountName(), namedBlobPath.getContainerName(),
-            namedBlobPath.getBlobName(), blobId, expirationTimeMs);
+            namedBlobPath.getBlobName(), blobId, expirationTimeMs, 0, properties.getBlobSize(), 0);
```

Comment:
please add a comment here that we don't need modified time passed to database, since it will be default to current_timestamp in database.

Reply:
Sure. Added comments